src/Slick.hs

```diff
@@ -1,13 +1,8 @@
-module Slick
-  (
-  -- * Slick
-  -- | This module re-exports everything you need to use Slick
-
-  -- ** Mustache
-    compileTemplate'
+-- | Slick
+--  This module re-exports everything you need to use Slick
 
-  -- ** Pandoc
-  , PandocReader
+module Slick
+  ( PandocReader
   , PandocWriter
   , markdownToHTML
```

Comment:
It might be good idea to add my code that allows to use arbitrary options for Pandoc to default example?

Reply:
Not sure which code you're referring here. Can you elaborate more?